landingai/io.py

```diff
@@ -10,11 +11,11 @@
 
 
 # TODO: support output type stream
-def read_file(url: str) -> bytes:
+def read_file(url: str) -> (bytes, str):
```

Comment:
this is a backward incompatible change.
how about we just parse the filename from the url?
(or maybe we need to think more about the underlying use case, love to learn more about it)

Reply:
Changed to Dict as discussed